src/gwt/acesupport/acemode/r_highlight_rules.js

```diff
@@ -227,6 +227,12 @@ define("mode/r_highlight_rules", ["require", "exports", "module"], function(requ
         regex : "#+(?!').*(?:----|====|####)\\s*$",
         next  : "start"
       },
+      {
+        // R Markdown chunk metadata comments
+        token : "comment.doc.tag",
+        regex : "#+[|].*$",
```

Comment:
It's just a single `#` that's supported for this syntax.

Reply:
I think the `comment.doc.tag` looks great here -- in nearly all doc themes it's identical to the comment itself, and in light themes it uses a shade of blue that is just a bit more recessive than comments which is great.

The one behavior we still need here is the termination of the metadata block: if you press `<Enter>` on a line with just `#|` it should wipe out the `#|` and leave a fully blank line in it's place (sort of like the behavior of bullets in most WYSIWYG editors)